addons/web/static/src/js/widgets/colorpicker.js

```diff
@@ -40,17 +40,19 @@ var ColorpickerWidget = Widget.extend({
         this.opacitySliderFlag = false;
         this.colorComponents = {};
 
-        var self = this;
-        var $body = $(document.body);
-        $body.on('mousemove.colorpicker', _.throttle(function (ev) {
-            self._onMouseMovePicker(ev);
-            self._onMouseMoveSlider(ev);
-            self._onMouseMoveOpacitySlider(ev);
-        }, 10));
-        $body.on('mouseup.colorpicker', _.throttle(function (ev) {
-            self.pickerFlag = false;
-            self.sliderFlag = false;
-            self.opacitySliderFlag = false;
+        const $document = $(document);
+        $document.on('mousemove.colorpicker', _.throttle((ev) => {
```

Comment:
Document instead of body?

Reply:
I would love to tell you why but I have not looked into it. When moving it from the modal to the left panel it no longer worked. document did the job and I don't see an issue with it so I went with that.

Reply:
No issue indeed, I was just curious as I don't know the difference. Maybe add a comment so that nobody (including me) changes it